int16 digit
geometry_msgs/Point[] points
---
bool end
